<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupportDirsDialog</class>
 <widget class="QDialog" name="SupportDirsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Previews</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePreviews"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnPreviews">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Titles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTitles"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnTitles">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hiscores</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leHiscores"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnHiscores">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Cheats</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCheats"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnCheats">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Samples</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSamples"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSamples">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>SupportDirsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>SupportDirsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
